src/pudl/metadata/resources/eia923.py

```diff
@@ -15,6 +15,7 @@
                 "fuel_mmbtu_per_unit",
                 "sulfur_content_pct",
                 "ash_content_pct",
+                "combined_heat_power",
```

Comment:
If this is truly a new field that we were inappropriately dropping (and losing entirely) before, and we want to have it independently associated with the boilers (in addition to the `associated_combined_heat_power` field that is applied to generators) then this seems appropriate.

Reply:
Okay, I think the point is that #509 is specific to generators. I hadn't been thinking about generators vs. boilers specifically. I'll do the analogous thing just for generators.

Reply:
This doesn't sound right to me. #509 (the issue) is very generic, even if it provides an example from the generators.

We're doing dropping potentially informational columns across a bunch of the EIA table transforms, and would prefer to feed them into the harvesting process so they can be used to more appropriately set the boiler / generator / utility / plant / balancing authority attributes (static or annual).

Addressing #509 should not have any effect on the database schema -- if there are columns in the wrong place, or columns we have failed to integrate entirely, those should probably be their own separate issues.  This issue should just stop dropping columns prior to the harvesting process, so they can be used within it.